EIPS/eip-1480.md

```diff
@@ -0,0 +1,186 @@
+---
+eip: 1480
```

Comment:
Where did this number came from? It seems you can choose from 1481 (the issue number) or 1550 (the PR number).